### PR TITLE
Chinese(Traditional) translation #402

### PR DESCRIPTION
**Pull request by jd615645**
I added Chinese(Traditional) language translation for harbor.
This version for Taiwanese wording habits.

### CODE REVIEW

**Comment by heww** [comment]
Please rebase the code with the latest master branch, thanks.

**Comment by jd615645** [comment]
ok, I already rebase the code with the latest master branch.

**Comment by heww** [comment]
Do you test to switch this language? I tested but not work.

**Comment by jd615645** [comment]
Sorry, I forgot commit language supporting list(supportedLangs).
Now it's fixed.

File: src/portal/src/app/base/navigator/navigator.component.html [hunk 1]
```diff
@@ -22,6 +22,7 @@
             <clr-dropdown-menu *clrIfOpen>
                 <a href="javascript:void(0)" clrDropdownItem (click)='switchLanguage("en-us")' [class.lang-selected]='matchLang("en-us")'>English</a>
                 <a href="javascript:void(0)" clrDropdownItem (click)='switchLanguage("zh-cn")' [class.lang-selected]='matchLang("zh-cn")'>中文简体</a>
+                <a href="javascript:void(0)" clrDropdownItem (click)='switchLanguage("zh-cn")' [class.lang-selected]='matchLang("zh-tw")'>中文繁體</a>
```
Comment (AllForNothing) [inline src/portal/src/app/base/navigator/navigator.component.html:25]: Please change 'switchLanguage("zh-cn")' to switchLanguage("zh-tw")

**Comment by jd615645** [comment]
Why don't merage?

**Comment by AllForNothing** [comment]
> Why don't merage?

@xaleeks Need your final comfirmation

**Pull request merged**